test/mocks/elasticsearch.mock.js

```diff
@@ -1,9 +1,9 @@
 "use strict";
 
 const sinon = require("sinon");
-const Elasticsearch = require("../../lib/service/storage/elasticsearch");
+const ElasticSearch = require("../../lib/service/storage/elasticsearch");
 
-class ElasticsearchMock extends Elasticsearch {
+class ElasticsearchMock extends ElasticSearch {
```

Comment:
I don't know if it's a good idea to update the Elasticsearch class name spelling